homeassistant/components/tractive/sensor.py

```diff
@@ -128,10 +132,22 @@ async def async_added_to_hass(self):
     TractiveSensorEntityDescription(
         key=ATTR_BATTERY_LEVEL,
         name="Battery Level",
-        native_unit_of_measurement=PERCENTAGE,
```

Comment:
The unit is necessery here.

Reply:
Why?
The unit for device_class battery is percentage, https://developers.home-assistant.io/docs/core/entity/sensor/#available-device-classes

Reply:
Because we don't have unit now

![image](https://user-images.githubusercontent.com/478555/131898282-00a64e24-da6c-4199-a974-7c69f3d0e680.png)
![image](https://user-images.githubusercontent.com/478555/131898322-fe6c86cd-bce1-4043-86a5-45fd7886e5a6.png)

Reply:
From documentation:
![image](https://user-images.githubusercontent.com/478555/131898595-d7e57f22-0ae7-4bfc-8427-f4aafec799da.png)